scripts/build.go

```diff
@@ -0,0 +1,61 @@
+// +build none
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strings"
+)
+
+func main() {
+
+	log.SetFlags(log.Lshortfile)
+
+	if _, err := os.Stat(filepath.Join("scripts", "build.go")); os.IsNotExist(err) {
+		log.Fatal("should run build from root dir")
+	}
+	if len(os.Args) < 2 {
+		log.Fatal("cmd required, eg: install")
+	}
+	switch os.Args[1] {
+	case "install":
+		install()
+	default:
+		log.Fatal("cmd not found", os.Args[1])
+	}
+}
+
+func install() {
+
+	argsList := append([]string{"list"}, []string{"./..."}...)
+
+	cmd := exec.Command(filepath.Join(runtime.GOROOT(), "bin", "go"), argsList...)
```

Comment:
Do we have a README.md that gives examples of a proper use? (ignore if this has been already raised)

Reply:
Good point, I'll update the readme.

Reply:
done